tests/syntax/test_classes.py

```diff
@@ -46,9 +35,9 @@ class Foo:\n
 def test_property_simple():
     scenario = compileScenic("""
         class Foo:
-            position: 3 @ 9
+            position: (3, 9)
```

Comment:
`@` needs #56, so changing it to tuples. Can revert the change once #56 is merged.